tests/unit/test_value.c

```diff
@@ -249,6 +249,7 @@ SENTRY_TEST(value_object_merge)
     TEST_CHECK_INT_EQUAL(sentry_value_as_int32(a), 1);
     TEST_CHECK_INT_EQUAL(sentry_value_as_int32(b), 20);
     TEST_CHECK_INT_EQUAL(sentry_value_as_int32(c), 30);
+
```

Comment:
huh, surely this commit can't be doing anything?  I only inserts two blank lines?